federation/schema.ipldsch

```diff
@@ -0,0 +1,20 @@
+type FederationHead  struct {
+    Head            Link
+    Topic  optional String
+    PublicKey       Bytes
+    Signature       Bytes
+}
+
+type FederationSnapshot struct {
+    Epoch             Int
+    VectorClock       Int
```

Comment:
This is this instances clock, not the full vector clock, right?

Reply:
Right